test/runtests.jl

```diff
@@ -8,6 +8,15 @@ using SparseArrays
 using DelimitedFiles
 using CSV
 
+@testset "Humann2" begin
+    df = CSV.read("files/humanntestfile.tsv", DataFrame, delim='\t', header=["#GeneFamily", "SRS014459-Stool_Abundance"], datarow = 2)
+    metadata = CSV.File("files/metadata_test_humann.csv") |> Tables.Array
```

Comment:
There aren't actually any tests here... should at least test the type and shape of the object created.

Reply:
I keep running into this error when I run `humann2_regroup `and `humann2_rename`

>   ERROR: IOError: could not spawn `humann2_rename_table -i /var/folders/jm/_25dbvt53_793g7w_1975c5c0000gn/T/jl_ahMrMf -n ec -o /var/folders/jm/_25dbvt53_793g7w_1975c5c0000gn/T/jl_afnvCf`: no such file or directory (ENOENT)

Do you know how I can resolve this?